internal/ipfamily/ipfamily.go

```diff
@@ -24,6 +24,9 @@ func ForAddresses(ips ...string) (Family, error) {
 	switch len(ips) {
 	case 1:
 		ip := net.ParseIP(ips[0])
+		if ip == nil {
```

Comment:
this was stolen from MetalLB, mind fixing it there too?

Reply:
done https://github.com/metallb/metallb/pull/2120